tools/pollers/overpass/childcare.overpassql

```diff
@@ -0,0 +1,13 @@
+[out:json][timeout:25];
+
+area["boundary"="administrative"]["name"="Vaterstetten"]->.boundingarea;
+
+(
+    nwr(area.boundingarea)["amenity"="kindergarten"];
+    nwr(area.boundingarea)["amenity"="childcare"];
+
+    nwr(area.boundingarea)["construction:amenity"="kindergarten"];
+    nwr(area.boundingarea)["construction:amenity"="childcare"];
```

Comment:
The information whether an element is still under construction is currently lost in the generated CSV. A new column "is construction" or similar probably needs to be added.